packages/flame/lib/src/components/component.dart

```diff
@@ -73,6 +73,14 @@ class Component {
   /// If two components share the same priority, they will probably be drawn in
   /// the order they were added.
   int get priority => _priority;
+  set priority(int newPriority) {
+    if (parent == null) {
+      _priority = newPriority;
+    } else {
+      parent?.children.changePriority(this, newPriority);
```

Comment:
`parent!`

Reply:
Shouldn't really matter which of the operators that are used, or is `!` a tiny bit cheaper because it skips a null check?